src/routes/+layout.svelte

```diff
@@ -61,41 +61,42 @@
 	}
 
 	async function initializeUserSessionData(setViewMode = 'admin') {
```

Comment:
After making the changes to ensure that the FB SDK is always loaded, I believe the changes to this function are not strictly necessary in order to fix #25, however, it's probably best not to call `logout` if the user isn't actually logged in (`$user == null`).